examples/creatives/banner_nonlinear.js

```diff
@@ -0,0 +1,61 @@
+class BannerNonLinear extends BaseSimidCreative{
+    constructor() {
+        super();
+        this.addButtonClickActions_();
+    }
+
+    /**
+     * Receives init message from the player.
+     * @param {!Object} eventData Data from the event.
+     * @protected
+     */
+    onInit(eventData) {
+        super.onInit(eventData);
+        this.dynamicResize_();
+    }
+
+    /**
+     * Adds actions to different buttons available on the overlay.
+     * @private
+     */
+    addButtonClickActions_() {
+        this.sendMessageOnButtonClick_("close_ad", CreativeMessage.REQUEST_STOP);
+        this.sendMessageOnButtonClick_("ad_text", CreativeMessage.REQUEST_EXPAND);
+        this.sendMessageOnButtonClick_("minimize_ad", CreativeMessage.REQUEST_COLLAPSE);
+    }
+
+    /**
+     * Listens for a click event on a button
+     * @param {String} elementName The name of the element.
+     * @param {String} message The message to send to the player.
+     * @private
+     */
+    sendMessageOnButtonClick_(elementName, message) {
+        const sendMessageFunction = () => {this.simidProtocol.sendMessage(message);}
+        document.getElementById(elementName).addEventListener(
+            'click', sendMessageFunction.bind(this));
```

Comment:
I think the anonymous function might not need a bind call.

Reply:
Good call, it doesn't. Fixed.